src/metabolic_ninja/celery.py

```diff
@@ -36,7 +36,7 @@
     # Our tasks are expected to be mostly cpu bound, so the number of concurrent
     # processes is set to match the number of cores available in the deployment
     # configuration, currently 4 vCPUs.
-    worker_concurrency=4,
+    worker_concurrency=int(os.environ['WORKER_CONCURRENCY']),
```

Comment:
I would have introduced the default here and not included the environment variable in all of the services above.

```suggestion
    worker_concurrency=int(os.environ.get('WORKER_CONCURRENCY', 4)),
```

Reply:
Hmm, I think I'll merge this as is, but feel free to remake this suggestion in a new PR.